CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.17.2
+
+**Release date:** 2021-11-04
+
```

Comment:
Can you add a short description of what this fixes for people?

Reply:
0.17.1 lacks a description too

Reply:
Did that because it was a nil pointer bug fix that didn't seem to require much of a description.

Reply:
I think all changelog entries should have a description.

Reply:
@darkowlzz can you add something like:

> This prerelease fixes a pointer error that was returned in `v0.17.0` during
> the import of public keys to verify a commit.

To `v0.17.1`?

Reply:
Added description for both the versions.